ethpm.json

```diff
@@ -18,10 +18,8 @@
     "voting"
   ],
   "dependencies": {
-    "tokens": "1.0.0",
     "dll": "1.0.4",
-    "attrstore": "1.0.0",
-    "zeppelin": "1.3.0"
+    "attrstore": "1.0.0"
```

Comment:
don't think we need `attrstore` or `dll` anymore .. in fact I think we can remove ethpm entirely